Sources/Lingo/DataSources/FileDataSource.swift

```diff
@@ -11,48 +15,34 @@ public final class FileDataSource: LocalizationDataSource {
     }
     
     // MARK: LocalizationDataSource
-    public func availableLocales() -> [LocaleIdentifier] {
-        do {
-            let identifiers = try FileManager().contentsOfDirectory(atPath: self.rootPath).filter {
-                $0.hasSuffix(".json")
-            }.map {
-                $0.components(separatedBy: ".").first! // It is safe to use force unwrap here as $0 will always contain the "."
-            }
-            
-            return identifiers
-
-        } catch let e {
-            assertionFailure("Failed retrieving contents of a directory: \(e.localizedDescription)")
-            return []
+    public func availableLocales() throws -> [LocaleIdentifier] {
+        return try FileManager().contentsOfDirectory(atPath: self.rootPath).filter {
+            $0.hasSuffix(".json")
+        }.map {
+            $0.components(separatedBy: ".").first! // It is safe to use force unwrap here as $0 will always contain the "."
```

Comment:
what happens, if the file name is `en.myLocalizedFile.json`?

Reply:
It would return `en` in that iteration.